rich/console.py

```diff
@@ -327,9 +327,10 @@ class Capture:
         console (Console): A console instance to capture output.
     """
 
-    def __init__(self, console: "Console") -> None:
+    def __init__(self, console: "Console", **kwargs) -> None:
```

Comment:
Don't see any reason to use kwargs here. It breaks typing.

Reply:
hmm guessing you're suggesting a default arg instead - `def __init__(..., echo: bool=True)`?